### ISSUE TITLE
Proposal: add Next.js to docs/examples

### ISSUE THREAD

**Issue by bravo-kernel** [tw-in-js/twind#43]
Are you open for a PR to add Next to https://github.com/tw-in-js/twind/blob/main/docs/examples.md ?

It took me a long time to figure out how to get this working in Next.js. The reason I got confused was because most React examples use below double quoted examples like shown below:

```js
    <main className="${tw`h-screen bg-purple-400 flex items-center justify-center`}">
      <h1 className="${tw`font-bold text(center 5xl white sm:gray-800 md:pink-700)`}">
        This is Twind!
      </h1>
    </main>
```

After removing the double quotes, things simply started working ❤️ 

```js
    <main className={tw`h-screen bg-purple-400 flex items-center justify-center`}>
      <h1 className={tw`font-bold text(center 5xl white sm:gray-800 md:pink-700)`}>
        This is Twind!
      </h1>
    </main>
```

**Comment by sastan** [comment]
I think you may have looked at the htm examples.... 😉 

PRs are always most welcome. We have a next.js example in the [SSR page](https://github.com/tw-in-js/twind/blob/main/docs/ssr.md#nextjs). Maybe that could be adopted or linked.

**Comment by lukejacksonn** [comment]
Yeh double quotes are only required on vanilla JS. If you use htm or JSX then you can omit the quotes. We put the vanilla examples up because we didn't want to be framework specific. Perhaps we should consider defaulting to (p)react snippets.

**Comment by bravo-kernel** [comment]
Addressed via #46
